react/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
There's a "node_modules/" in sirepo/.gitignore. Perhaps that should be node_modules (no /). There's a package-lock.json in sirepo/.gitignore so don't think it is needed here. I think this file doesn't need to exist iow.